checks/dangerous_workflow.go

```diff
@@ -81,8 +81,10 @@ type triggerName string
 
 var (
 	triggerPullRequestTarget = triggerName("pull_request_target")
+	triggerWorkflowRun        = triggerName("workflow_run")
 	triggerPullRequest       = triggerName("pull_request")
-	checkoutUntrustedRef     = "github.event.pull_request"
+	checkoutUntrustedPullRequestRef     = "github.event.pull_request"
```

Comment:
why didn't they add checkoutUntrustedRef for pull_request_target too?

Reply:
The "ref" is related to the source which is "pull_request" and not to the action trigger which is "pull_request_target"